src/components/ConversationSettings/ConversationAvatarEditor.vue

```diff
@@ -210,9 +246,38 @@ export default {
 			}
 		},
 
+		setEmoji(emoji) {
+			this.emojiAvatar = emoji
+		},
+
 		saveAvatar() {
 			this.showCropper = false
 			this.loading = true
+
+			if (this.emojiAvatar) {
+				this.saveEmojiAvatar()
+			} else {
+				this.savePictureAvatar()
+			}
+		},
+
+		async saveEmojiAvatar() {
+			try {
+				await this.$store.dispatch('setConversationEmojiAvatarAction', {
+					token: this.conversation.token,
+					emoji: this.emojiAvatar,
+					color: this.backgroundColor ? this.backgroundColor.slice(1) : null,
+				})
+				this.loading = false
+				this.emojiAvatar = ''
+				this.backgroundColor = ''
+			} catch (e) {
+				showError(t('spreed', 'Error saving profile picture'))
+				this.loading = false
+			}
```

Comment:
`setConversationEmojiAvatarAction` doesn't throw any exceptions. 

Also, there it is not a `profile picture`, but a `conversation picture`.